.github/workflows/dev_adbc.yml

```
@@ -55,7 +55,7 @@ jobs:
         with:
           path: ~/conda_pkgs_dir
           key: conda-${{ runner.os }}-${{ steps.get-date.outputs.today }}-${{ env.CACHE_NUMBER }}-${{ hashFiles('ci/**') }}
-      - uses: conda-incubator/setup-miniconda@v3
+      - uses: conda-incubator/setup-miniconda@e5457fe695e3787d415a42160e580fd154de1899  # v3
```

It seems that the revision doesn't exist: https://github.com/conda-incubator/setup-miniconda/commit/e5457fe695e3787d415a42160e580fd154de1899

```suggestion
      - uses: conda-incubator/setup-miniconda@505e6394dae86d6a5c7fbb6e3fb8938e3e863830  # v3.1.1
```